### PR TITLE
Feature/user signup and login

### PR DESCRIPTION
**Pull request by karidreyer**

### CODE REVIEW

File: src/components/login-view/login-view.jsx [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+                    type="password"
+                    value={password}
+                    onChange={(e) => setPassword(e.target.value)}
+                    required
+                />
+            </label>
+            <button type="submit">Submit</button>
+        </form>
+    );
+};
```
Comment (Stanley-Okwii) [inline src/components/login-view/login-view.jsx:68]: Could you please add propTypes here.

File: src/components/main-view/main-view.jsx [hunk 2]
```diff
@@ -2,14 +2,26 @@ import { useState, useEffect } from "react";
...
+        if (!token) {
+            return;
+        }
+
+        fetch("https://movie-nest-app-630a7e8ce836.herokuapp.com/movies", {
+            headers: { Authorization: `Bearer ${token}`}
+        })
           .then((response) => response.json())
           .then((data) => {
             console.log("movies from api:", data);
```
Comment (Stanley-Okwii) [inline src/components/main-view/main-view.jsx]: console.logs are great for debugging but kindly remove them in the final submission.

**Review by Stanley-Okwii — APPROVED** [review]
Hi @karidreyer ,

Great work 🎉, I left a couple of comments to help tidy up your work.

File: src/components/login-view/login-view.jsx (hunk 1 above)
Comment (karidreyer) [reply src/components/login-view/login-view.jsx:68]: Added. Thanks for catching that. :)

File: src/components/main-view/main-view.jsx (hunk 2 above)
Comment (karidreyer) [reply src/components/main-view/main-view.jsx]: console.log removed

**Pull request merged**